jobs/app-jobs.yml

```diff
@@ -88,6 +88,13 @@
     jobs:
       - 'generic-template'
 
+- project:
+    name: email-notifier
```

Comment:
this should match the github repo name

Reply:
(which is confusing, because it doesn't match the maven artifact, or what I usually say when I talk about the project; sorry)

Reply:
Done! Updated it here and in ecr_config repository. Thank you for letting me know.